apps/api/infra/api.ts

```diff
@@ -190,6 +190,11 @@ export const serviceSetup = (services: {
         staging: 'https://api-staging.thinglysing.is/business/tolfraedi',
         prod: 'https://api.thinglysing.is/business/tolfraedi',
       },
+      FORM_SYSTEM_API_BASE_PATH: {
+        dev: 'https://profun.island.is/umsoknarkerfi',
+        staging: '',
```

Comment:
Are only going to test this on dev? 

What will happen if this gets deployed on staging and prod? Broken pod?

Reply:
for the time being this will only be tested on dev, I was told to leave staging and prod empty

Reply:
I tested locally to start the api with empty path and it started, so should be fine.